osmosis/assetlist.json

```diff
@@ -26406,11 +26406,11 @@
           "counterparty": {
             "chain_name": "aaronetwork",
             "base_denom": "uaaron",
-            "channel_id": "channel-6"
+            "channel_id": "channel-7"
           },
           "chain": {
-            "channel_id": "channel-91254",
-            "path": "transfer/channel-91254/uaaron"
+            "channel_id": "channel-91942",
+            "path": "transfer/channel-91942/uaaron"
```

Comment:
![image](https://github.com/user-attachments/assets/fe62f2e3-b33d-43b2-b90b-0dc7aa00ffbc)

Reply:
> Typically, IBC channels between mainnets cannot be changed once registered (without governance approval or proof that there has been no token transfer). There is already some amount of tokens transferred to Osmosis using the old channel (albeit, just a small dust amount). I recognize your GitHub account and presume the purpose is likely valid, but what is the purpose of this change?

Our trust period has expired. Since all channel operations were test ones, there were no token trades, we decided to recreate the channel.

This is the address of our fund (https://ping.pub/Aaron%20Network/account/aaron140qa42t5y60rmdvnvlarmqwuanth837v9n70f9):

<img width="833" alt="Screenshot 2024-12-28 at 06 09 24" src="https://github.com/user-attachments/assets/56d06d6a-889f-41f2-a5c2-ebd7942f2adb" />

This is the developer's address (osmo1rfz9nsprthegf43jytuatvgjwmv54zx6pscu88 => cosmos1rfz9nsprthegf43jytuatvgjwmv54zx6fttv34). The team's addresses are listed on our GitHub https://github.com/aaronetwork/airdrop-data

<img width="832" alt="Screenshot 2024-12-28 at 06 21 30" src="https://github.com/user-attachments/assets/5b4542f6-1d01-4d89-be1c-3af5d23357af" />

These changes will not affect our users.

To avoid such situations (expired trust period), we have developed a monitoring service:

<img width="527" alt="Screenshot 2024-12-28 at 06 44 23" src="https://github.com/user-attachments/assets/4cf7948c-3883-41c0-a9de-173223ea554c" />